### PR TITLE
Fix Cleanroom logic with an LV energy hatch

### PR DESCRIPTION
**Pull request by Caedis**
Fixes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/15473

### CODE REVIEW

**Comment by HoleFish** [comment]
What if a ulv energy hatch is installed?

**Comment by HoleFish** [comment]
Cleanroom logic got broken with a ulv energy hatch too.

**Comment by Caedis** [comment]
With the latest commit
image

File: src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_Cleanroom.java [hunk 1]
```diff
@@ -124,8 +125,21 @@ public String[] getStructureDescription(ItemStack itemStack) {
     @Override
     public CheckRecipeResult checkProcessing() {
         mEfficiencyIncrease = 100;
+        final long inputVoltage = getMaxInputVoltage();
+
+        // only allow LV+ energy hatches
+        if (inputVoltage < 32) {
```
Comment (OneEyeMaker) [inline src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_Cleanroom.java:131]: Please use corresponding constant from `TierEU` class (here and below).

Comment (Caedis) [reply src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_Cleanroom.java:131]: Didnt know that was a thing

**Review by OneEyeMaker — APPROVED** [review]

**Comment by Caedis** [comment]
Thought I spotlessed

**Pull request merged**